pyseir/inference/infer_rt.py

```diff
@@ -468,6 +475,10 @@ def infer_all(self, plot=False, shift_deaths=0):
                     for col in df_all.columns:
                         if timeseries_type.value in col:
                             df_all[col] = df_all[col].shift(shift_in_days)
+                            # Extend death and hopitalization rt signals beyond shift to avoid sudden jumps in composit metric.
```

Comment:
Do you have a sense of how often this will happen? Would adding a logging.debug call with 'we interpolated X points' just swamp any possible usefulness? I have haven't seen any instances where we have feature specific loggers, so I'm guessing this might swamp anyone's reasonable expectation of the general debug logger.

Reply:
This will happen for all death series, since the timeshifting is almost always bigger than 0. I'll log it though.